lib/season.rb

```diff
@@ -1,6 +1,66 @@
 class Season
-  def initialize(game_team_data, team_data)
+  def initialize(game_team_data, game_data)
     @game_team_data = game_team_data
-    @team_data = team_data
+    @game_data = game_data
   end
-end
\ No newline at end of file
+
+  def coach_totals(season)
```

Comment:
Coach Totals and Coach Wins could be broken off into modules for later refactoring.

Reply:
Agree, this is a great opportunity to clean this up. We'll just need to remember to unit test the helpers.

Reply:
Tests have been included.